lms/static/scripts/frontend_apps/services/client-rpc.js

```diff
@@ -1,11 +1,30 @@
 import { Server, call as rpcCall } from '../../postmessage_json_rpc';
+import { apiCall } from '../utils/api';
+import { JWT } from '../utils/jwt';
 
 /**
  * @typedef User
  * @prop {string} displayName
  * @prop {string} userid
  */
 
+/**
+ * @typedef ServiceConfig
+ * @prop {string} grantToken
+ */
+
+/**
+ * The subset of the Hypothesis client configuration that `ClientRpc` references.
+ *
+ * The backend will set other configuration which is just forwarded to the client
+ * and not touched by `ClientRpc`.
+ *
+ * See https://h.readthedocs.io/projects/client/en/latest/publishers/config/.
+ *
+ * @typedef ClientConfig
+ * @prop {[ServiceConfig]} services
```

Comment:
The fact that `services` is an array is part of the client's configuration: https://h.readthedocs.io/projects/client/en/latest/publishers/config/#cmdoption-arg-services

The reason this is typed as an array of one item is because in the LMS context we know that this array will always be generated on the backend with one entry. I'll add a comment to clarify.